.github/workflows/main-build.yml

```diff
@@ -93,3 +96,27 @@ jobs:
           TEST_LOG_ANALYTICS_WORKSPACE_ID: ${{ secrets.TEST_LOG_ANALYTICS_WORKSPACE_ID }}
           TEST_STORAGE_CONNECTION_STRING: ${{ secrets.TEST_STORAGE_CONNECTION_STRING }}
         run: make e2e-test
+
+  validate-arm64:
+    needs: build
+    name: validate-arm64
+    runs-on: ARM64
```

Comment:
Shouldn't we add `concurrency` here? I know it is running in a container, but not sure how many parallel executions the machine could handle. But maybe it is just and overthinking.

Reply:
good question! I think no because each runner runs one job in parallel and smoke tests don't use any external service